specs/binding-mqtt-kafka.spec/src/main/resources/META-INF/zilla/mqtt_kafka.idl

```diff
@@ -40,7 +40,6 @@ scope mqtt_kafka
 
         struct MqttKafkaResetMqttConnectionEx extends core::stream::Extension
         {
-            string16 reason;
```

Comment:
This event type structure name is out of date. Let's rename it to `MqttKafkaNonCompactSessionsTopicEx`, for consistency with other components.

Then let's also add the topic name used by the binding when recording the event, and include that topic name in the output when formatting the message.

Reply:
Updated, please review the new wording with the added `topic` field.